sdk/storage/Azure.Storage.Queues/tests/QueueClientTests.cs

```diff
@@ -775,6 +830,53 @@ public async Task CanSendAndReceiveNonUTFBytes()
             CollectionAssert.AreEqual(content, receivedMessage.Body.ToArray());
         }
 
+        [Test]
+        public async Task FailsOnInvalidQueueMessageIfNoHandlerIsProvided()
+        {
+            // Arrange
+            await using DisposingQueue test = await GetTestQueueAsync();
+            var encodingClient = GetEncodingClient(test.Queue.Name, QueueMessageEncoding.Base64);
+            var nonEncodedContent = "test_content";
+
+            await test.Queue.SendMessageAsync(nonEncodedContent);
+
+            // Act
+            await TestHelper.AssertExpectedExceptionAsync<FormatException>(
+                encodingClient.ReceiveMessagesAsync(),
+                e =>
+                {
+                    StringAssert.Contains("The input is not a valid Base-64 string", e.Message);
+                });
+        }
+
+        [Test]
+        public async Task CanHandleInvalidMessageAndReturnValid()
```

Comment:
Can you, please, add a test with multiple subscriptions to the event?

Reply:
Done.